pandas/core/indexes/datetimes.py

```diff
@@ -78,21 +78,26 @@ def _new_DatetimeIndex(cls, d):
 )
 class DatetimeIndex(DatetimeTimedeltaMixin):
     """
-    Immutable ndarray of datetime64 data, represented internally as int64, and
-    which can be boxed to Timestamp objects that are subclasses of datetime and
-    carry metadata such as frequency information.
+    Immutable ndarray of datetime64 data.
+
+    Represented internally as int64, and which can be boxed to Timestamp objects
```

Comment:
its a stretch to say "represented internally as int64" is accurate.

mentioning that DTI has freq is good, but id rather avoid mentioning it for Timestamp.